artemis-core-client/src/main/java/org/apache/activemq/artemis/core/config/FederationConfiguration.java

```diff
@@ -74,7 +74,7 @@ public Map<String, FederationPolicy> getQueuePolicies() {
    }
 
    // strange spelling!, it allows a type match for singular of correct plural Policies from properties
-   public FederationConfiguration addQueuePolicie(FederationQueuePolicyConfiguration federationPolicy) {
+   public FederationConfiguration addQueuePolicy(FederationQueuePolicyConfiguration federationPolicy) {
```

Comment:
Comment above method is inaccurate after the name changes.

Reply:
removed the old comment